### ISSUE TITLE
Bypass MDM and using location?

### ISSUE THREAD

**Issue by robertjulis** [assafdori/bypass-mdm#10]
Hello, 

I coped with the task quite well, everything was simple and I thank the author for creating this instruction. However, I have a question that I can't find the answer to on the web. Is it safe to use the location on the MacBook after using the bypass?

**Comment by ccjcan** [comment]
hi @assafdori  thanks for this great work. Question, when you say Apple would brick it, does that mean the computer wouldn't be able to start / be used ?

**Comment by assafdori** [comment]
> hi @assafdori thanks for this great work. Question, when you say Apple would brick it, does that mean the computer wouldn't be able to start / be used ?

It's not really an issue unless the company who bought it has registered it with the inventory system of Apple, which I'm not really sure how it's called, but unless the company really puts effort with Apple to brick the device, you should be good. For example, a stolen company laptop. But it takes effort, time and resources to get Apple to brick it, so I wouldn't worry.

**Comment by ccjcan** [comment]
> > hi @assafdori thanks for this great work. Question, when you say Apple would brick it, does that mean the computer wouldn't be able to start / be used ?
> 
> It's not really an issue unless the company who bought it has registered it with the inventory system of Apple, which I'm not really sure how it's called, but unless the company really puts effort with Apple to brick the device, you should be good. For example, a stolen company laptop. But it takes effort, time and resources to get Apple to brick it, so I wouldn't worry.

Sorry another question. Will Apple OS updates overwrite your code's changes?

**Comment by assafdori** [comment]
> > > hi @assafdori thanks for this great work. Question, when you say Apple would brick it, does that mean the computer wouldn't be able to start / be used ?
> > 
> > 
> > It's not really an issue unless the company who bought it has registered it with the inventory system of Apple, which I'm not really sure how it's called, but unless the company really puts effort with Apple to brick the device, you should be good. For example, a stolen company laptop. But it takes effort, time and resources to get Apple to brick it, so I wouldn't worry.
> 
> Sorry another question. Will Apple OS updates overwrite your code's changes?

No, you should not be prompted to configure the MDM again, unless you factory reset.

**Comment by ahmedmahershaaban** [comment]
Can I update the operating system if there is a new one ? Or will it lock for me again ?

**Comment by assafdori** [comment]
> Can I update the operating system if there is a new one ? Or will it lock for me again ?

You can update freely without issues.

**Comment by mateussiqueira** [comment]
The bypass does not affect Location Services. You can use Find My, Maps, etc. normally after bypassing MDM.

The only blocked domains are:
- `deviceenrollment.apple.com`
- `mdmenrollment.apple.com`
- `iprofiles.apple.com`

iCloud, Find My, iMessage, and Software Update are **not** blocked (v3/unleash explicitly leaves `gdmf.apple.com` and `albert.apple.com` unblocked).
